From 066f9537e41fec8491946f45f1965717803dc98b Mon Sep 17 00:00:00 2001
From: Alexander Bluhm <alexander.bluhm@gmx.net>
Date: Sat, 20 Mar 2021 15:43:20 +0100
Subject: [PATCH] Use $arg to match @ARGV in Makefile.PL

As the default pattern match was used, $arg was not evaluated.  This
fixes warning "Use of uninitialized value $_ in pattern match (m//)
at Makefile.PL line 19".
---
 Makefile.PL | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.PL b/Makefile.PL
index 42fca50..eaf47e0 100644
--- Makefile.PL
+++ Makefile.PL
@@ -16,7 +16,7 @@ use Config qw(%Config);
 my %cfg;
 @cfg{qw(cc ccflags ldflags)} = @Config{qw(cc ccflags ldflags)};
 for my $arg (@ARGV) {
-  if ( /^(CC|CCFLAGS|LDFLAGS)=(.*)/i ) {
+  if ( $arg =~ /^(CC|CCFLAGS|LDFLAGS)=(.*)/i ) {
     $cfg{lc($1)} = $2;
   }
 }
